src/nbl/builtin/utils.cmake

```diff
@@ -8,14 +12,20 @@ macro(LIST_BUILTIN_RESOURCE _BUNDLE_NAME_ _LBR_PATH_)
 	set(_LBR_${_BUNDLE_NAME_}_ ${_LBR_${_BUNDLE_NAME_}_} PARENT_SCOPE) # override
 endmacro()
 
-function(ADD_CUSTOM_BUILTIN_RESOURCES _TARGET_NAME_ _BUNDLE_NAME_ _BUNDLE_SEARCH_DIRECTORY_ _NAMESPACE_ _OUTPUT_DIRECTORY_)
+# Creates a library with builtin resources for given bundle
+# _TARGET_NAME_ is name of a target library that will be created
+# _BUNDLE_NAME_ is a bundle name builtin resources are associated with
```

Comment:
you mean a list variable populated using `LIST_BUILTIN_RESOURCE`?

Reply:
resolved